go/cmd/dolt/commands/blame_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import "testing"
+
+func TestTruncateString(t *testing.T) {
```

Comment:
This test (and the function it's testing) should maybe live somewhere else; not sure where yet, there are still corners of the project structure I'm not familiar with.